src/components/Molecules/widgets/ChatBotWidget/index.tsx

```
@@ -0,0 +1,230 @@
+  const [isTyping, setIsTyping] = React.useState(false);
+
+  const { refs, context } = useFloating({
+    open: isOpen,
+    onOpenChange: setIsOpen,
+  });
+
+  const click = useClick(context);
+  const role = useRole(context);
+  const { getReferenceProps } = useInteractions([click, role]);
+
+  const handleAddBotMessage = React.useCallback((message: string) => {
```

ESLint

[error] 'message' is defined but never used.

(@typescript-eslint/no-unused-vars)